pkg/golinters/lll.go

```diff
@@ -89,6 +90,8 @@ func getLLLIssuesForFile(filename string, maxLineLen int, tabSpaces string) ([]r
 	lineNumber := 0
 	multiImportEnabled := false
 
+	urlComment := regexp.MustCompile(`\s*//\s*http(s)?://[^ ]+$`)
```

Comment:
Would it be possible to also exclude URLs used as reference links? e.g.

```markdown
Here's a [link] and [another][1]!

[link]: https://example.com
[1]: https://example.com
```

I something use these references to keep Go docs readable (inserting long URLs makes following text hard), and given modern godoc uses markdown, this would likely be universally helpful.